<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="chooseFileBtn"/>
   <addaction name="separator"/>
   <addaction name="saveImageBtn"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="flipHorizontallyBtn"/>
   <addaction name="separator"/>
   <addaction name="flipVerticallyBtn"/>
   <addaction name="separator"/>
   <addaction name="flipHorizontallyVerticallyBtn"/>
   <addaction name="separator"/>
  </widget>
  <action name="chooseFileBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/dir.png</normaloff>:/images/icons/dir.png</iconset>
   </property>
   <property name="text">
    <string>Choose file</string>
   </property>
   <property name="toolTip">
    <string>Choose file</string>
   </property>
  </action>
  <action name="saveImageBtn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/save.png</normaloff>:/images/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save image</string>
   </property>
   <property name="toolTip">
    <string>Save image</string>
   </property>
  </action>
  <action name="flipHorizontallyBtn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/flip-horizontal.png</normaloff>:/images/icons/flip-horizontal.png</iconset>
   </property>
   <property name="text">
    <string>Flip horizontally</string>
   </property>
   <property name="toolTip">
    <string>Flip horizontally</string>
   </property>
  </action>
  <action name="flipVerticallyBtn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/flip-vertical.png</normaloff>:/images/icons/flip-vertical.png</iconset>
   </property>
   <property name="text">
    <string>Flip vertically</string>
   </property>
   <property name="toolTip">
    <string>Flip vertically</string>
   </property>
  </action>
  <action name="flipHorizontallyVerticallyBtn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/flip-horizontal-vertical.png</normaloff>:/images/icons/flip-horizontal-vertical.png</iconset>
   </property>
   <property name="text">
    <string>Flip horizontally and vertically</string>
   </property>
   <property name="toolTip">
    <string>Flip horizontally and vertically</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
